plugins/ci/skills/fetch-tide-pr-status/fetch_tide_pr_status.py

```
@@ -0,0 +1,268 @@
+
+        inc = tq.get("includedBranches", [])
+        exc = tq.get("excludedBranches", [])
+        if branch in exc:
+            continue
+        if inc and branch not in inc:
+            continue
+
+        required = sorted(tq.get("labels", []), key=len)
+        forbidden = sorted(tq.get("missingLabels", []), key=len)
+
+        labels = [{"name": l, "have": l in pr_labels_set} for l in required]
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)